x-pack/plugins/security_solution/public/management/pages/host_isolation_exceptions/view/hooks.ts

```diff
@@ -120,7 +120,7 @@ export function useFetchHostIsolationExceptionsList({
         filter: kql,
       });
     },
-    { enabled }
+    { enabled, keepPreviousData: true }
```

Comment:
Keep old data while performing a new search

Reply:
👍 

I'm assuming we should always use this, right?